src/App.js

```diff
@@ -1,4 +1,6 @@
 import React, {Component} from 'react';
+import io from 'socket.io-client';
```

Comment:
Ah, maybe this is why you don't need a script tag for this. I like your approach better!